js/app.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+var itemListArray = [];
+
+//////////////////////////////////////////
+////////////// Constructor
+//////////////////////////////////////////
+
+function Item(itemName, itemPath){
+  this.itemName       = itemName;
+  this.itemPath       = itemPath;
+  this.itemShownTotal = 0;
+  itemListArray.push(this);
+  this.numberOfTimesClicked = 0;
+}
+
+var totalClicks = 0;
+
+////////////// Individual items
+//////////////////////////////////////////
+var a = new Item ('bag', 'assets/bag.jpg');
+var b = new Item ('banana', 'assets/banana.jpg');
+var c = new Item ('bathroom', 'assets/bathroom.jpg');
+var d = new Item ('boots', 'assets/boots.jpg');
+var e = new Item ('breakfast', 'assets/breakfast.jpg');
+var f = new Item ('bubblegum', 'assets/bubblegum.jpg');
+var g = new Item ('chair', 'assets/chair.jpg');
+var h = new Item ('chtulu', 'assets/cthulhu.jpg');
+var i = new Item ('dog-duck', 'assets/dog-duck.jpg');
+var j = new Item ('dragon', 'assets/dragon.jpg');
+var k = new Item ('pen', 'assets/pen.jpg');
+var l = new Item ('pet-sweep', 'assets/pet-sweep.jpg');
+var m = new Item ('scissors', 'assets/scissors.jpg');
+var n = new Item ('shark', 'assets/shark.jpg');
+var o = new Item ('sweep', 'assets/sweep.png');
+var p = new Item ('tauntaun', 'assets/tauntaun.jpg');
+var q = new Item ('unicorn', 'assets/unicorn.jpg');
+var r = new Item ('usb', 'assets/usb.gif');
+var s = new Item ('water-can', 'assets/water-can.jpg');
+var t = new Item ('wine glass', 'assets/wine-glass.jpg');
+//old items will fill previouslyShownUserPageArray, currentlyShownUserPageArray will //compare to previouslyShownUserPageArray and have different results. itemListArray will hold list of all items
+
+function randomItemSelectionFunc(){
+  return Math.floor(Math.random() * (itemListArray.length));
+};
+
+var previouslyShownUserPageArray  = [];
+
+function randomPictureGenerator(){
+  var currentlyShownUserPageArray   = [];
+  console.log('previous' + previouslyShownUserPageArray);
+  while(currentlyShownUserPageArray.length < 3){
+    var randomItemSelectionVar = randomItemSelectionFunc();
+    if(!previouslyShownUserPageArray.includes(randomItemSelectionVar) && !currentlyShownUserPageArray.includes(randomItemSelectionVar)){
+      currentlyShownUserPageArray.push(randomItemSelectionVar);
+    }
+  }
+  previouslyShownUserPageArray = currentlyShownUserPageArray;
+  var imageLeft   = itemListArray[currentlyShownUserPageArray[0]];//.itemPath;
```

Comment:
Indents should be a single space for submitted work,  if multiple spaces to align assignments helps for dev work, that's fine but for assignment submissions reduce to a single space.  This is just industry standards and the more code you write the easier it will be to track / see.